build.rs

```diff
@@ -18,6 +17,9 @@ fn main() {
              please consider using the `std` cargo feature instead\""
         );
     }
+    if env::var("CARGO_FEATURE_ALIGN").is_ok() {
+        println!("cargo:warning=\"libc's align cargo feature is deprecated (and always available)");
```

Comment:
(Just an aside: `cargo:warning` messages are [only emitted for local/path dependencies](https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargo-warning) if `-vv` is not used, so these warnings are unlikely to ever be seen.)

Reply:
That's a good point, thank you.